airflow-core/tests/unit/api_fastapi/common/db/test_dags.py

```diff
@@ -52,11 +54,17 @@ def setup_teardown(self):
     def dag_with_queued_run(self, session):
         """Returns a DAG with a QUEUED DagRun and null start_date."""
 
+        bundle_name = "testing"
+        orm_dag_bundle = DagBundleModel(name=bundle_name)
```

Comment:
Can't this fixture also use the 'testing_dag_bundle' fixture?

Reply:
will update